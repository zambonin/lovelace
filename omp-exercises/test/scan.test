#!/bin/sh

./omp-exercises/iterative_scan 10 +
