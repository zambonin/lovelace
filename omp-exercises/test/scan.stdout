3 4 7 9 13 17 20 25 29 33 
